### PR TITLE
Moves temporary files into 'tmp' folder

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@redhat.com>

This PR moves temporary files into 'tmp' folder. It is useful in the case of using debug key:

```
docker run --rm -t \
       -v ${PWD}/:/workspace/project  \
       quay.io/che-incubator/dash-licenses:next --debug
```

### CODE REVIEW

File: src/entrypoint.sh [hunk 1]
```diff
@@ -71,13 +67,17 @@ if [ ! -d $DEPS_DIR ]; then
     echo
 fi
 
+mkdir -p $PROJECT_COPY_DIR
```
Comment (sleshchenko) [inline src/entrypoint.sh:70]: please consider moving it inside `Copy project...` block

File: src/package-manager/yarn/bump-deps.js [hunk 2]
```diff
@@ -21,28 +19,30 @@ const {
...
 } = require('../../document.js');
 
-const EXCLUSIONS_DIR = join(__dirname, `project/${DEPS_DIR}/EXCLUDED`);
+const args = process.argv.slice(2);
+let writeToDisk = true;
+if (args[1] === '--check') {
+  writeToDisk = false;
+}
 
+const DEPS_DIR = args[0];
```
Comment (sleshchenko) [inline src/package-manager/yarn/bump-deps.js:28]: Why you need deps dir as argument? isn't it always the same?

File: src/entrypoint.sh (hunk 1 above)
Comment (olexii4) [reply src/entrypoint.sh]: I have fixed it.

File: src/package-manager/yarn/bump-deps.js (hunk 2 above)
Comment (olexii4) [reply src/package-manager/yarn/bump-deps.js]: I have fixed it.

**Review by akurinnoy — APPROVED** [review]

**Review by sleshchenko — APPROVED** [review]
Thanks for addressing comments

**Pull request merged**